### PR TITLE
Reverted Issue/1

### PR DESCRIPTION
**Pull request by AmanKala**
Made the changes as per the feedbacks received in the previous PR (#2 ).
For a full description kindly refer to #2.

### CODE REVIEW

**Review by vijay-singh07 — APPROVED** [review]
looks nice to me.

**Review by vyombangwal — APPROVED** [review]
LGTM

File: routes/web.php [hunk 1]
```diff
@@ -16,15 +16,15 @@
...
-Route::view('/login','login');
+Route::view('/login','login')->name('login');
 Route::post('login',[AuthController::class,'check']);
 
 Route::get('/logout', [AuthController::class,'logout']);
 
-Route::group(['middleware'=>['AuthCheck']],function(){
-    Route::get('/', [AuthController::class,'dashboard']);   
+Route::group(['middleware'=>['auth.check']],function(){
+    Route::get('/', [AuthController::class,'dashboard'])->name('/');
```
Comment (Gauravbhatt19) [inline routes/web.php]:
```suggestion
    Route::get('/', [AuthController::class,'dashboard'])->name('dashboard');   
```
Always use relevant, and specific route names.

File: app/Http/Controllers/AuthController.php [hunk 2]
```diff
@@ -4,28 +4,20 @@
...
 
         // Saving the values in database.
-        $user=new User;
-        $user->first_name=$req->first_name;
-        $user->last_name=$req->last_name; 
-        $user->email=$req->email;
-        $user->password=$req->password;
-        $user->save();
-        return redirect('login');
+        User::create($req->all());
```
Comment (Gauravbhatt19) [inline app/Http/Controllers/AuthController.php]:
```suggestion
        User::create($req->validated());
```

File: app/Http/Controllers/AuthController.php [hunk 3]
```diff
@@ -4,28 +4,20 @@
...
     {
-        $req->validate([
-            "first_name" => "required",
-            "last_name" => "required",
-            "email" => "required|email",
-            "password" => "required|min:6",
-            "re_enter_password" => "required|same:password"
-        ]);
+        //To Ensure that the entered data follow the defined set of rules.
+        $req->validated();
```
Comment (Gauravbhatt19) [inline app/Http/Controllers/AuthController.php]:
```suggestion
```
unnecessary as on line:18 this will return the array of validated data

**Review by Gauravbhatt19 — APPROVED** [review]
Minor suggestions, rest looks good to merge 👍🏻

**Review by ishgary — APPROVED** [review]

**Pull request merged**